README.md

```diff
@@ -14,13 +18,29 @@ Please send an email to security@filecoin.org. See our [security policy](SECURIT
 
 ## Development
 
-All work is tracked via issues. An attempt at keeping an up-to-date view on remaining work is in the [lotus testnet github project board](https://github.com/filecoin-project/lotus/projects/1).
-
 The main branches under development at the moment are:
 * [`master`](https://github.com/filecoin-project/lotus): current testnet.
 * [`next`](https://github.com/filecoin-project/lotus/tree/next): working branch with chain-breaking changes.
 * [`ntwk-calibration`](https://github.com/filecoin-project/lotus/tree/ntwk-calibration): devnet running one of `next` commits.
 
+### Tracker
+
+All work is tracked via issues. An attempt at keeping an up-to-date view on remaining work is in the [lotus github project board](https://github.com/orgs/filecoin-project/projects/8).
+
+### Packages
+
+The lotus Filecoin implementation unfolds into the following packages:
+
+- [This repo](https://github.com/filecoin-project/lotus)
+- [lotus-docs](https://github.com/filecoin-project/lotus-docs)
+- [go-fil-markets](https://github.com/filecoin-project/go-fil-markets)
+- [go-storage-market](https://github.com/filecoin-project/go-storage-market)
+- [go-storage-miner](https://github.com/filecoin-project/go-storage-miner)
```

Comment:
This repo isn't used
```suggestion
```

Reply:
Thank you @magik6k. Can I update the repo as deprecated then?

Reply:
Not 100% sure, it may be used by GFC